apps/app.go

```diff
@@ -11,11 +11,13 @@ package apps
 
 import (
 	"github.com/gorilla/mux"
+	"github.com/heketi/heketi/pkg/glusterfs/api"
 	"net/http"
 )
 
 type Application interface {
 	SetRoutes(router *mux.Router) error
+	TopologyInfo() (*api.TopologyInfoResponse, error)
```

Comment:
I understand what you are trying to do here, but I'm not in love with altering the Application interface. I'm going to think on it a little more and perhaps come up with a more concrete suggestion.

Reply:
I felt it was not worth blocking this change for. Since I couldn't come up with an better approach I could articulate I figured it was something we could improve later. Probably with changes to how our url endpoints are registered.